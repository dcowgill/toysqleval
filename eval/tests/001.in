create table test (n integer, f number);
insert into test values (1, 0.99999);
insert into test values (2, 2.71828);
insert into test values (3, 3.14159);
select * from test;
select n from test;
select f from test;
select n*f, f*n, f/n from test;
