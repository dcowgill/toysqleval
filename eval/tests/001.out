OK
OK
OK
OK
 n | f      
---+---------
 1 | 0.99999
 2 | 2.71828
 3 | 3.14159
 n
---
 1
 2
 3
 f      
---------
 0.99999
 2.71828
 3.14159
 ?                 | ?                 | ?                 
-------------------+-------------------+--------------------
 0.99999           | 0.99999           | 0.99999           
 5.43656           | 5.43656           | 1.35914           
 9.424769999999999 | 9.424769999999999 | 1.0471966666666666
